resources/prometheus/unit_tests/RHACSFleetshardSyncCentralReconcilePaused.yaml

```diff
@@ -12,18 +12,18 @@ tests:
       - eval_time: 1h
         alertname: RHACSFleetshardSyncCentralReconcilePaused
         exp_alerts: []
-      - eval_time: 30h
```

Comment:
You need to add more values to the test series, otherwise there is no data at `eval_time`.